src/DemoAppDelegate.swift

```diff
@@ -36,6 +36,7 @@ class DemoAppDelegate: NSObject, UIApplicationDelegate {
             .setOauthAuthorizationParamsProvider { OAuthAuthorizationParameters(
                 scopes: Configuration.scopes
             ) }
+            .setTokenBoundConfiguration(configuration: BoundedTokenConfiguration())
```

Comment:
The 4.x SDK was missing this setting which led to an error when using the 4.x SDK with the following client setting, which protects the refresh token:

- Issue Token Bound Authorization Code

Therefore I had not updated the SDK in a while. Generally, the inability for mobile clients to protect refresh tokens is an issue for mobile apps. Therefore we want to have that option enabled.